pages/eco_news_page.py

```
@@ -0,0 +1,75 @@
+import allure
+
+from pages.base_page import BasePage
+from pages.locators.eco_news_page_locators import *
```

Ruff (0.12.2)

`from pages.locators.eco_news_page_locators import *` used; unable to detect undefined names

(F403)